### PR TITLE
Added plugin config to allow or disallow meeting recordings

### PR DESCRIPTION
**Pull request by harshilsharma63**

### CODE REVIEW

**Comment by harshilsharma63** [comment]
Fixes #82

File: server/helpers.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func (p *Plugin) PopulateMeeting(m *dataStructs.MeetingRoom, details []string, d
 	m.AttendeePW_ = "ap"
 	m.ModeratorPW_ = "mp"
 	m.Record = "true"
```
Comment (moschlar) [inline server/helpers.go:63]: Why don't you set this to the config value, too?

Comment (harshilsharma63) [reply server/helpers.go:63]: @moschlar great question. As per [the docs](https://docs.bigbluebutton.org/dev/api.html#create), the `record` flag requires manually starting the recording to work. It won't matter setting its value from config. Do you think it has an effect?

Comment (moschlar) [reply server/helpers.go:63]: I thought that when `record=true`, the BBB server always records everything, and in post-processing, the recording will be processed and published or deleted based on the existence of start stop markers.
So your change would basically always make BBB record and discard it afterwards, which would be a waste of resources that would be mitigated by `record=false`.

Comment (harshilsharma63) [reply server/helpers.go:63]: Hmm, that makes sense. I'll update for the same.

Comment (harshilsharma63) [reply server/helpers.go:63]: Done. This is now available in [v4.0.2](https://github.com/blindsidenetworks/mattermost-plugin-bigbluebutton/releases/tag/v4.0.2)

**Pull request merged**